source/tutorial/autocomplete-component.md

```diff
@@ -368,90 +374,95 @@ Since our component is expecting the filter process to be asynchronous, we retur
 waits for all asynchronous tasks to complete before running the given function callback.
 It returns a promise that we also return from the test.
 
 If you return a promise from a QUnit test, the test will wait to finish until that promise is resolved.
-In this case our test completes when the `wait` helper decides that processing is finished,
+In this case our test completes when the `settled` helper decides that processing is finished,
 and the function we provide that asserts the resulting state is completed.
 
 ```tests/integration/components/list-filter-test.js{+3,+33,+34,+35,+36}
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, settled } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
-import wait from 'ember-test-helpers/wait';
-import RSVP from 'rsvp';
+import { resolve } from 'rsvp';
 
 const ITEMS = [{city: 'San Francisco'}, {city: 'Portland'}, {city: 'Seattle'}];
 const FILTERED_ITEMS = [{city: 'San Francisco'}];
 
-moduleForComponent('list-filter', 'Integration | Component | filter listing', {
-  integration: true
-});
-
-test('should initially load all listings', function (assert) {
-  // we want our actions to return promises, since they are potentially fetching data asynchronously
-  this.on('filterByCity', () => {
-    return RSVP.resolve({ results: ITEMS });
-  });
+module('Integration | Component | rental-listing', function(hooks) {
+  setupRenderingTest(hooks);
 
-  // with an integration test,
-  // you can set up and use your component in the same way your application will use it.
-  this.render(hbs`
-    {{#list-filter filter=(action 'filterByCity') as |results|}}
-      <ul>
-      {{#each results as |item|}}
-        <li class="city">
-          {{item.city}}
-        </li>
-      {{/each}}
-      </ul>
-    {{/list-filter}}
-  `);
+  test('should initially load all listings', async function (assert) {
+    // we want our actions to return promises, since they are potentially fetching data asynchronously
+    this.on('filterByCity', () => {
+      return RSVP.resolve({ results: ITEMS });
+    });
 
-  return wait().then(() => {
-    assert.equal(this.$('.city').length, 3);
-    assert.equal(this.$('.city').first().text().trim(), 'San Francisco');
+    // with an integration test,
+    // you can set up and use your component in the same way your application will use it.
+    await render(hbs`
+      {{#list-filter filter=(action 'filterByCity') as |results|}}
+        <ul>
+        {{#each results as |item|}}
+          <li class="city">
+            {{item.city}}
+          </li>
+        {{/each}}
+        </ul>
+      {{/list-filter}}
+    `);
+
+    return settled().then(() => {
+      assert.equal(this.element.querySelectorAll('.city').length, 3);
+      assert.equal(this.element.querySelector('.city').textContent().trim(), 'San Francisco');
```

Comment:
`qunit-dom` just made it into ember-cli's default blueprints, so should we start mentioning it in the guides too? Because anyone who reads these changes going forward will see it in their newly generated app. We can always add a note mentioning that it comes recommended & that people should start using it even in their old apps after using the [codemod](https://github.com/simplabs/qunit-dom-codemod)

Reply:
yes, we should update the guides to use it by default. it has the nice side-effect of making the assertions much less noisy.

Reply:
super-rentals is on 3.0, and the blueprints aren't qunit dom, is it in beta?

Reply:
It's on master so guessing it'll be out with 3.1

Reply:
we'll circle back and update then